docs/examples/rag.md

```diff
@@ -4,12 +4,12 @@ RAG search example. This demo allows you to ask question of the [logfire](https:
 
 Demonstrates:
 
-* [retrievers](../agents.md#retrievers)
+* [tools](../agents.md#tools)
 * [agent dependencies](../dependencies.md)
 * RAG search
 
 This is done by creating a database containing each section of the markdown documentation, then registering
-the search tool as a retriever with the PydanticAI agent.
+the search tool as a tool with the PydanticAI agent.
```

Comment:
is the double reference to the word tool here okay? I think it might be but not exactly sure what "the search tool" is referring to here or if it might be confusing. (I assume it's fine but just want you to confirm.)